tests/bloaty_test_pe.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test.h"
+
+struct BloatyTestEntry
+{
+  std::string name;
+  std::vector<std::string> commandline;
+  std::string input_file;
+  std::string result_file;
+};
+
+std::string TestEntryName(const testing::TestParamInfo<struct BloatyTestEntry>& entry) {
+  return entry.param.name;
+}
+
+std::ostream& operator<<(std::ostream& os, const BloatyTestEntry& entry) {
+  os << "{ ";
+  for (const auto& str: entry.commandline) {
+    os << str << ", ";
+  }
+  os << entry.input_file << ", " << entry.result_file << " }";
+  return os;
+}
+
+// Strip all trailing whitespace (including \r)
+void Normalize(std::string& contents) {
```

Comment:
Added this `Normalize` function, that is used on both files read, and the bloaty output.